tutorcredentials/plugin.py

```diff
@@ -19,78 +19,67 @@
 # CONFIGURATION
 ########################################
 
+config: t.Dict[str, t.Dict[str, t.Any]] = {
+    "defaults": {
+        "VERSION": __version__,
+        "BACKEND_SERVICE_EDX_OAUTH2_PROVIDER_URL": "http://lms:8000/oauth2",
+        "BACKEND_SERVICE_EDX_OAUTH2_KEY": "{{ CREDENTIALS_OAUTH2_KEY }}",
+        "DOCKER_IMAGE": "{{ DOCKER_REGISTRY }}overhangio/openedx-credentials:{{ CREDENTIALS_VERSION }}",
+        "EXTRA_PIP_REQUIREMENTS": [],
+        "FAVICON_URL": "https://edx-cdn.org/v3/default/favicon.ico",
+        "HOST": "credentials.{{ LMS_HOST }}",
+        "LOGO_TRADEMARK_URL": "https://edx-cdn.org/v3/default/logo-trademark.svg",
+        "LOGO_TRADEMARK_URL_PNG": "https://edx-cdn.org/v3/default/logo-trademark.png",
+        "LOGO_TRADEMARK_URL_SVG": "https://edx-cdn.org/v3/default/logo-trademark.svg",
+        "LOGO_URL": "",
+        "LOGO_URL_PNG": "{{ CREDENTIALS_LOGO_URL }}",
+        "LOGO_URL_SVG": "",
+        "LOGO_WHITE_URL": "{{ CREDENTIALS_LOGO_URL }}",
+        "LOGO_WHITE_URL_PNG": "{{ CREDENTIALS_LOGO_URL }}",
+        "LOGO_WHITE_URL_SVG": "",
+        "MYSQL_DATABASE": "credentials",
+        "MYSQL_USERNAME": "credentials",
+        "OAUTH2_KEY": "credentials-key",
+        "OAUTH2_KEY_DEV": "credentials-key-dev",
+        "OAUTH2_KEY_SSO": "credentials-key-sso",
+        "OAUTH2_KEY_SSO_DEV": "credentials-key-sso-dev",
+        "PLATFORM_NAME": "{{ PLATFORM_NAME }}",
+        "PRIVACY_POLICY_URL": "{{ LMS_HOST }}/privacy-policy",
```

Comment:
`"{% if ENABLE_HTTPS %}https{% else %}http{% endif %}://{{ LMS_HOST }}/privacy-policy"`

Reply:
I think that this PR needs to be rebased on top of master. @CodeWithEmad you seem to be missing some of the latest changes. Sorry!

Reply:
Yes @regisb, you are right.

Reply:
Oh, I missed that one. 

> I think that this PR needs to be rebased on top of master

It's Done.

Reply:
Actually `PRIVACY_POLICY_URL` was removed in the latest code of main branch.
Can u please rebase it again or sync it with master as `PRIVACY_POLICY_URL` is still in the PR.
Thanks for bearing it with me 👍

Reply:
It's Done.